js_modules/dagit/packages/core/src/instigation/InstigationTick.tsx

```diff
@@ -137,22 +134,29 @@ export const RunList: React.FC<{
   );
 
   if (loading || !data) {
-    return <Spinner purpose="section" />;
+    return (
+      <Box padding={32}>
+        <Spinner purpose="section" />
+      </Box>
+    );
   }
 
   if (data.pipelineRunsOrError.__typename !== 'Runs') {
     return (
-      <NonIdealState
-        icon="error"
-        title="Query Error"
-        description={data.pipelineRunsOrError.message}
-      />
+      <Box padding={32}>
```

Comment:
Might be nice to include the padding in the Spinner/NonIdealState since I think we do this at a lot of callsites.